backend/.env.template

```diff
@@ -5,6 +5,7 @@ JWT_SECRET=$JWT_SECRET
 JWT_EXPIRES_IN=$JWT_EXPIRES_IN
 GRAPHIQL=false
 GDT_API_URL=$GDT_API_URL
+ENV_NAME=$ENV_NAME
```

Comment:
I understand the idea, but I have serious doubts about this.
 1. We do not protect production from simply setting: `SEND_EMAIL=false`
 2. The admin doing the deploy should be able to switch of `EMAIL_TEST_MODUS`

Reply:
deleted